blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -57,10 +57,12 @@ const HeaderAccountAction = ({ customFields }) => {
   }, [Identity, loggedIn]);
 
   const handleLogout = (e) => {
-    if (e.key === 'Enter') {
+    if (e.key === 'Enter' || e.type === 'click') {
+      e.preventDefault();
       Identity.logout().then(() => {
         setIsLoggedIn(false);
         setUser(null);
```

Comment:
could we remove the ui setState. since we're navigating off the page, it's kind of moot
```suggestion
```